cmd/gossamer/commands/utils.go

```diff
@@ -481,3 +482,55 @@ func setViperDefault(config *cfg.Config) {
 		}
 	}
 }
+
+func parseLogLevel() error {
+	// set default log level
+	moduleToLogLevel := map[string]string{
+		"global":  cfg.DefaultLogLevel,
+		"core":    cfg.DefaultLogLevel,
+		"digest":  cfg.DefaultLogLevel,
+		"sync":    cfg.DefaultLogLevel,
+		"network": cfg.DefaultLogLevel,
+		"rpc":     cfg.DefaultLogLevel,
+		"state":   cfg.DefaultLogLevel,
+		"runtime": cfg.DefaultLogLevel,
+		"babe":    cfg.DefaultLogLevel,
+		"grandpa": cfg.DefaultLogLevel,
+		"wasmer":  cfg.DefaultLogLevel,
+	}
+
+	if logLevel != "" {
+		logConfigurations := strings.Split(logLevel, ",")
+		for _, config := range logConfigurations {
+			parts := strings.SplitN(config, "=", 2)
+			if len(parts) != 2 {
+				return fmt.Errorf("Invalid log configuration: %s", config)
+			}
```

Comment:
nit: GoLand complains
```suggestion
			if len(parts) != 2 {
				return fmt.Errorf("invalid log configuration: %s", config)
			}
```